### PR TITLE
Prepare initial changelog for v2.0.0 release.

### PR DESCRIPTION
**Pull request by zach-klippenstein**

### CODE REVIEW

**Comment by zach-klippenstein** [comment]
Holding off on this while we rethink the API.

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+Version 2.0.0
+-------------
+
+_2020-08-11_
+
+* First public release!
+* Convert the code to Kotlin. (#8, #9)
+* Rewrite the API to be more Kotlin-friendly. (#16)
+* Introduce `StateRenderer` to allow custom rendering of arbitrary view attributes. (#23)
```
Comment (zach-klippenstein) [inline CHANGELOG.md:12]: Need to confirm this matches whatever name we end up with in #23.

**Comment by zach-klippenstein** [comment]
This should be more or less ready to land once #23 is merged.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+* Drop the `com.squareup` package prefix, so the package is now just `radiography`. (#15)
+* Add a sample app. (#18)
+* Fix: Crash when `skippedIds` is empty. (#16)
+* Fix: `FocusedWindowViewFilter` wasn't filtering correctly. (#18)
+
+Version starts at 2.0.0 to differentiate from some internal releases.
```
Comment (pyricau) [inline CHANGELOG.md:18]: and because 2.0 is always a more mature release! ;)

File: CHANGELOG.md (hunk 1 above)
Comment (zach-klippenstein) [reply CHANGELOG.md:12]: It does.

**Comment by zach-klippenstein** [comment]
Updated for alpha release.

**Pull request merged**